src/kit/Card.module.scss

```diff
@@ -66,3 +66,11 @@
     min-width: 100%;
   }
 }
+.dark {
+  @extend %elevation-1-dark;
```

Comment:
These are incorrect. The elevation needs to always be one level higher than whatever the card sits on. Please consult with @EmilyBonar.